pages.fr/common/vagrant.md

```diff
@@ -0,0 +1,36 @@
+# vagrant
+
+> Gère des environnements de développement légers, reproductibles et portables.
+> Plus d’informations : <https://www.vagrantup.com>.
```

Comment:
```suggestion
> Plus d'informations : <https://www.vagrantup.com>.
```

https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/more-info-link.md#fr

This will solve this bot check:

> The page pages.fr/common/vagrant.md has a more info link that does not match the one in the English page. Please check the "More information:" translation as well using https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/more-info-link.md.

The link is already the same.